Doc/library/os.rst

```diff
@@ -2292,6 +2292,8 @@ features:
    will fail with an :exc:`OSError` subclass in a number of cases:
```

Comment:
I think it's better to keep this sentence since explicitly stating that all the related exceptions below are subclasses of `OSError` could be helpful for easily catching the exceptions, but I can remove it nonetheless if wanted here.